### PR TITLE
Platform force function expressions

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?

Disallows the use of function declarations.

## What is the value of this and can you measure success?

Our JS is more succinct as it allows the use of arrow functions and implicit returning. This avoids un-needed function and return keywords. Forcing a particular style also ensures consistency.

**Note:** This currently doesn't auto`--fix` unfortunately. Not sure if it's not possible to do for some reason...

## Does this affect other platforms - Amp, Apps, etc?

No.

## Tested in CODE?

No.

### CODE REVIEW

**Review by gu-stav — APPROVED** [review]
Looks good!

File: static/src/javascripts/.eslintrc.js [hunk 1]
```diff
@@ -61,5 +61,6 @@ module.exports = {
         // our own rules for frontend
         // live in tools/eslint-plugin-guardian-frontend
         'guardian-frontend/global-config': 2,
+        'func-style': ['error', 'expression', { allowArrowFunctions: true }],
```
Comment (sndrs) [inline static/src/javascripts/.eslintrc.js]: this should inherit from the top level one automatically?

**Review by sndrs — APPROVED** [review]
this must have been fun :)

lgtm, apart from comment

File: static/src/javascripts/.eslintrc.js (hunk 1 above)
Comment (gtrufitt) [inline static/src/javascripts/.eslintrc.js]: Oh yeah, I thought I was in javascripts

**Comment by gtrufitt** [comment]
> this must have been fun :)

Kinda was! Really quick when you get into a routine. Would defo be nicer if there was a `--fix` for it though!

**Pull request merged**